src/Main.java

```diff
@@ -1,5 +1,71 @@
+import java.time.LocalDate;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        task1();
+        task2();
+        task3();
     }
+    public static void task1(){
+        System.out.println("Задание 1");
+        int year = 1700;
+        if (year < 1584){
+            System.out.println("Данные заведены не корректно");
+        }
+        String leapYear = highYearCalculation(year);
+        System.out.println(year + leapYear);
+    }
+    public static String highYearCalculation(int year){
+        if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0 && year > 1584)){
```

Comment:
```suggestion
        if (year > 1584 && year % 4 == 0 && year % 100 != 0 || year % 400 == 0){
```
условие можно сократить, также скобки не нужны